spk/minio/Makefile

```diff
@@ -1,38 +1,30 @@
 SPK_NAME = minio
-SPK_VERS = 2020-11-25T22-36-25Z
-SPK_REV = 8
+SPK_VERS = 2021.09.18
+SPK_REV = 9
 SPK_ICON = src/minio.png
 DSM_UI_DIR = app
 
-DEPENDS = cross/busybox cross/$(SPK_NAME)
+DEPENDS = cross/$(SPK_NAME)
 
 UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
 MAINTAINER = fgma
 DESCRIPTION = MinIO is a high performance object storage server compatible with Amazon S3 APIs.
-RELOAD_UI = yes
 DISPLAY_NAME = MinIO
-CHANGELOG = "Major release, multiple feature and bugfix releases."
+CHANGELOG = "Update to MinIO 2021-06-14T01:29:23Z. Major release, multiple feature and bugfix releases."
 HOMEPAGE = https://min.io/
-LICENSE = Apache License 2.0
-STARTABLE = yes
+LICENSE = AGPLv3.0
```

Comment:
due to the license change we might need to include a license file with `LICENSE_FILE = ...` to include the license in the installation wizard.

Reply:
Fixed in https://github.com/SynoCommunity/spksrc/pull/4683/commits/3500ca31f987db7386b4cde11e9fc347cdd727a5